src/Page/Guide/View.elm

```diff
@@ -54,3 +55,8 @@ viewRelatedGuideTeasers guideList =
 
     else
         text ""
+
+guideTitle: List Style
+guideTitle =
+    [ fontFamilies [ "Ludicrous", "serif" ]
```

Comment:
Could argue for this to go in global with the other font declarations but f it only ever gets used here then this is a good home.

Reply:
My thinking is that this is some placeholder code to indicate how this particular font can be applied to a title. As it only appears on one page on the site then its okay to have it locally in one view, I think.